src/time_tracker/core.clj

```diff
@@ -31,4 +32,4 @@
   (condp = (first args)
```

Comment:
Prefer `case` over `condp = ` for matching against compile-time constants.

Reply:
@jysandy Providing the reason for this is essential rather than just making the statement. [This article](https://puredanger.github.io/tech.puredanger.com/2010/05/30/learning-clojure-15-case/) provides the reason for making the suggested change.